docker-compose.yml

```diff
@@ -4,29 +4,19 @@ services:
   rosie:
     build:
       context: rosie
-    volumes:
-      - /tmp/serenata-data:/tmp/serenata-data
 
   django:
     image: datasciencebr/jarbas-backend
-    environment:
-      - ALLOWED_HOSTS=localhost,127.0.0.1
-      - AMAZON_S3_BUCKET=serenata-de-amor-data
-      - AMAZON_S3_REGION=s3-sa-east-1
-      - AMAZON_S3_CEAPTRANSLATION_DATE=2016-08-08
-      - CELERY_BROKER_URL=amqp://guest:guest@queue/
-      - NEW_RELIC_APP_NAME=Jarbas (Django); Jarbas (Combined)
     depends_on:
       - elm
       - tasks
-    volumes:
-      - "./contrib/data:/mnt/data"
+    environment:
+      - NEW_RELIC_APP_NAME=Jarbas (Django); Jarbas (Combined)
     hostname: django
 
   tasks:
     image: datasciencebr/jarbas-backend
     environment:
-      - CELERY_BROKER_URL=amqp://guest:guest@queue/
       - NEW_RELIC_APP_NAME=Jarbas (Celery); Jarbas (Combined)
```

Comment:
As far as I got the idea of this PR, `NEW_RELIC_APP_NAME` should be in the env files, right?

Reply:
This is a special case where the value changes from one service to another.

```
NEW_RELIC_APP_NAME=Jarbas (Django); Jarbas (Combined)
NEW_RELIC_APP_NAME=Jarbas (Celery); Jarbas (Combined)
```

Reply:
True story :cocktail: